cdap-distributions/src/hdinsight/pkg/install.sh

```diff
@@ -26,7 +26,7 @@ CDAP_BRANCH='release/5.0'
 # otherwise, it should be CDAP_TAG=''
 CDAP_TAG=${CDAP_TAG:+hdi5.0}
 # The CDAP package version passed to Chef
-CDAP_VERSION='5.1.0-1'
+CDAP_VERSION='5.1.2-1'
```

Comment:
same

Reply:
Can we be sure to document this, I think it has come up many times

Reply:
We've been following the simple "set it to the next release" rule.  I assume develop will be version bumped to 5.1.3-SNAPSHOT?  In that case this should be 5.1.3.  Whether it happens now in this merge pr, or in a subsequent version bump PR doesn't matter (but i would suggest version bump PR, keeping this consistent with the rest of the branch).

Reply:
nvm, looks like develop branch is already 6.0.0-SNAPSHOT so 6.0.0